docs/source/upgrade-guide/index.md

```diff
@@ -509,6 +509,26 @@ The `react/jsx-key` rule has been enabled in ESlint for catching missing `key` i
 You might catch some violations in your project or add-on code after running ESlint.
 Adding the missing `key` property whenever the violation is reported will fix it.
 
+### `@plone/registry` moved to ESM
+
+The `@plone/registry` package has been moved to ESM.
+This unfortunately forces some import paths changes that should be patched in your Plone project or add-on boilerplates.
+
+```{note}
+As always that something changes in the boilerplate, you have the option to regenerate one from Cookieplone and move your code into it instead of fiddling with it.
```

Comment:
```suggestion
As always, when something changes in the boilerplate, you may regenerate one from Cookieplone and move your code into it, instead of fiddling with it.
```